operators/test-data/raster/dataset_defs/modis_ndvi.json

```diff
@@ -0,0 +1,27 @@
+{
+    "time": {
+        "time_intervals": [
+            {
+                "start": 0,
+                "end": 1
+            }
+        ]
+    },
+    "tile": null,
```

Comment:
Can't we just remove this information?

Reply:
the serialization test failed without it :(